Web/starter_project_barbies/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { BsFacebook, BsLinkedin, BsInstagram } from 'react-icons/bs'
+import Image from "next/image";
+
+type TeamCardProps = {
```

Comment:
i would recommend defining a member object and taking that as a prop than taking each one by one

so then you can have a list of member objects in the container and pass them into this component in a loop

Reply:
and i would also rename the component as team member card instead of team card

Reply:
> 

I did that on the team members page. Should I change it?

Reply:
> i would recommend defining a member object and taking that as a prop than taking each one by one
> 
> so then you can have a list of member objects in the container and pass them into this component in a loop

I did that on the team members page. Should I change it?

Reply:
you can rename it to TeamMemberCard
and the team members page i assume would be team-members